beerlog/beerlogdb.py

```diff
@@ -125,6 +125,22 @@ def GetScoreBoard(self):
     )
     return query
 
+  def GetGlassFromName(self, name):
+    """Returns the corresponding glass from a uid
+
+    Args:
+      name(str): the character name.
+    Returns:
+      int: the glass size for a character, or the default value if not found.
+    """
+
+    entry = Entry.select(Entry.amount).where(
```

Comment:
Not sure if using `get` would change something.

Reply:
sure